include/envoy/stats/store.h

```diff
@@ -66,6 +67,13 @@ class StoreRoot : public Store {
    */
   virtual void setTagProducer(TagProducerPtr&& tag_producer) PURE;
 
+  /**
+   * Attach a StatsMatcher to this StoreRoot to prevent the initialization of stats according to
+   * some ruleset.
+   * @param a StatsMatcher to attach to this StoreRoot.
```

Comment:
@param stats_matcher a StatsMatcher to attach to this StoreRoot.

Reply:
Done.